rollup.config.js

```diff
@@ -46,6 +46,6 @@ export default [
       }),
       prod && terser()
     ],
-    external: ['fetch']
+    external: ['node-fetch']
```

Comment:
How would this behave with people using other flavours of fetch?

Reply:
This config tells rollup not to include `node-fetch` in the bundle. Since this is the bundle that is targeting browsers, `window.fetch` will be used internally by node-fetch. The only thing we need to take care of is the compatibility with the node version so it works on both sides. List of differences is here: https://github.com/node-fetch/node-fetch/blob/master/docs/v2-LIMITS.md

Since this plugin previously used `cross-fetch` which internally uses `node-fetch` we know we're good.
The only change I made from the preview implem in this PR is that I included node-fetch in the node bundle since I felt it's not convenient and should not be the responsibility of a node user to include a fetch polyfill. So now it fixes a change that would have been breaking on node side, keeping a lighter bundle on browser side. Hope all this makes sense ;)

Reply:
Makes perfect sense, thanks a lot for your explanation